src/agent/rustjail/src/cgroups/systemd/common.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright 2021-2022 Kata Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+pub const DEFAULT_SLICE: &str = "system.slice";
+pub const SLICE_SUFFIX: &str = ".slice";
+pub const SCOPE_SUFFIX: &str = ".scope";
+pub const UNIT_MODE: &str = "replace";
+
+pub type Properties<'a> = Vec<(&'a str, zbus::zvariant::Value<'a>)>;
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub enum CgroupHierarchy {
+    Legacy,
+    Unified,
```

Comment:
How about just V1 and V2 to be future proof? `Unified` can be legacy in future as well.

Reply:
`legacy` and `unified` seem to be widely used labels to distinguish cgroups, e.g. cgroup v2 can be enabled by append `systemd.unified_cgroup_hierarchy=1` to boot options.
At present, `FsManager` of kata-agent uses legacy and unified to identify v1 and v2, and the same is true for runtime.